docs/administering/active-directory.md

```diff
@@ -0,0 +1,304 @@
+      way, you could change a user's e-mail address in the future without requiring that they
+      transfer all of their Sandstorm data to a new account. However, most administrators choose to
+      use e-mail addresses as the primary identifier, accepting that this means that e-mail
+      addresses can never change.
+
+![Screenshot of this step](https://alpha-evgl4wnivwih0k6mzxt3.sandstorm.io/windows-server-ad-config/15.png)
+
+- Click "OK" to finish editing claim rules.
+
+![Screenshot of this step](https://alpha-evgl4wnivwih0k6mzxt3.sandstorm.io/windows-server-ad-config/16.png)
+
+
+
+### Part 2: Extract your certificate
+
+- Open the AD FS management UI. In the left sidebar, browse to: "Service > Certificates".
+
+![Screenshot of this step](https://alpha-evgl4wnivwih0k6mzxt3.sandstorm.io/windows-server-ad-config/17.png)
+
+- Click on the certificate under "Token-signing" to select it.
+
+![Screenshot of this step](https://alpha-evgl4wnivwih0k6mzxt3.sandstorm.io/windows-server-ad-config/18.png)
+
+- Click "View Certificate..." In the Certificate dialog, go to the "Details" tab.
+
+![Screenshot of this step](https://alpha-evgl4wnivwih0k6mzxt3.sandstorm.io/windows-server-ad-config/19.png)
+
+- In the "Details" tab, click "Copy to File..."
+
+![Screenshot of this step](https://alpha-evgl4wnivwih0k6mzxt3.sandstorm.io/windows-server-ad-config/20.png)
+
+- The Certificate Export Wizard appears. Click "Next".
+
+![Screenshot of this step](https://alpha-evgl4wnivwih0k6mzxt3.sandstorm.io/windows-server-ad-config/21.png)
+
+- Choose the radio button for: "Base-64 encoded X.509 (.CER)". Click "Next."
+
+![Screenshot of this step](https://alpha-evgl4wnivwih0k6mzxt3.sandstorm.io/windows-server-ad-config/22.png)
+
+- Choose where to save the certificate. Click "Next."
+
+![Screenshot of this step](https://alpha-evgl4wnivwih0k6mzxt3.sandstorm.io/windows-server-ad-config/23.png)
+
+- Review the "Completing the Certificate Export Wizard" information, then click "Finish."
+
+![Screenshot of this step](https://alpha-evgl4wnivwih0k6mzxt3.sandstorm.io/windows-server-ad-config/24.png)
+
+- Open the certificate file using a text editor, such as Notepad. You will need to copy/paste this next in the next part.
+
+![Screenshot of this step](https://alpha-evgl4wnivwih0k6mzxt3.sandstorm.io/windows-server-ad-config/25.png)
+
+### Part 3: Configure Sandstorm
+
+- Open Sandstorm's SAML login configuration.
+
+    - Under "SAML provider entry point URL", enter your AD FS login entry point. This is usually
+      "/adfs/ls" at the host associated with your Active Directory server. For example, if your Active
+      Directory server is ad.example.com, the URL would be: https://ad.example.com/adfs/ls
+
+    - Under "SAML cert for above provider", copy the contents of the certificate you extracted in part
+      2, above.
+
+    - Use the full server URL as the entity ID. For example, https://example.sandcats.io
```

Comment:
As mentioned elsewhere, it's important that people *don't* change the Entity ID at this point, otherwise they'll break the config that has already been synced into AD FS. The default entity ID does not include "https://" and works just fine here.